README.md

```diff
@@ -26,15 +26,27 @@ pixabay.authenticate('username', 'api_key');
 // Set default query parameters to make with every request
 pixabay.defaults({safesearch: 'true'});
 
-// Create a request promise
-var requestPromise = pixabay
+// Create a request handler
+var handler = pixabay
     .request()
     .query({order: 'latest'})
-    .search(['dogs', 'puppies'])
-    .get();
+    .search(['dogs', 'puppies']);
+
+// Get the request promise
+var request = handler.get();
```

Comment:
This usage example seems a little clunky. Why not do:

```javascript
var requestPromise = pixabay.request()
  .query({order: 'latest'})
  .search(['dogs', 'puppies'])
  .get();  // Returns the request promise
```

This way we can completely abstract away the concept of a "request handler".